### PR TITLE
Add package.json

### PR DESCRIPTION
**Pull request by jellejurre**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "url": "https://vrlabs.dev"
+  },
+  "siteUrl": "https://github.com/VRLabs/Collision-Detection",
+  "vpmDependencies": {
+    "dev.vrlabs.instancer": ">=1.0.2"
+  },
+  "legacyFolders": {
+    "Assets\\VRLabs\\Collision Detection": "1b7ffa2cfaa4355428a147133e387cd1"
+  },
+  "unityPackageDestinationFolder": "Assets/VRLabs/Contact Tracker",
```
Comment (Cibbi) [inline package.json:20]: wrong folder

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "vccRepoCategory": "components",
+  "media": {
+    "previewImage": "https://raw.githubusercontent.com/VRLabs/Collision-Detection/main/Media/Web/Preview.webp",
+    "previewGif": "https://raw.githubusercontent.com/VRLabs/Collision-Detection/main/Media/Web/PreviewGif.webp"
+  },
+  "unityPackageDestinationFolderMetas": {
+    "Assets/VRLabs": "652a1ba5b00554143bc9a76307dbc4e8",
+    "Assets/VRLabs/Contact Tracker": "1b7ffa2cfaa4355428a147133e387cd1"
```
Comment (Cibbi) [inline package.json:28]: Also wrong folder

**Review by Cibbi — CHANGES_REQUESTED** [review]
I guess you copied it over the contact tracker.

Are the guids right? I see they're different than the contact tracker at least

**Comment by jellejurre** [comment]
Yes the guids are right

**Review by Cibbi — APPROVED** [review]
looks good to me

**Review by hfcRed — APPROVED** [review]

**Pull request merged**